content/sensu-go/6.1/operations/deploy-sensu/deployment-architecture.md

```diff
@@ -30,10 +30,14 @@ See [Run a Sensu cluster][7] and the [etcd documentation][4] for more informatio
 
 ## Hardware sizing
 
-Because etcd's design prioritizes consistency across a cluster, the speed with which write operations can be completed is very important to the performance of a Sensu cluster. 
+Because etcd's design prioritizes consistency across a cluster, the speed with which write operations can be completed is very important to the performance and health of a Sensu cluster. 
```

Comment:
```suggestion
Because etcd's design prioritizes consistency across a cluster, the speed with which write operations can be completed is very important to the Sensu cluster's performance and health.
```